lisa/messages.py

```diff
@@ -168,6 +190,38 @@ class DiskPerformanceMessage(PerfMessage):
     randwrite_iops: Decimal = Decimal(0)
     randwrite_lat_usec: Decimal = Decimal(0)
 
+    # Sysbench FileIO params
+    read_io_per_sec: Decimal = Decimal(0)
```

Comment:
Please try best to reuse above fields, like `read_io_per_sec` should be `read_iops` above. So the data from different tools can be compared together.

Reply:
Done

Reply:
Please check all other fields too.

1. Make sure it's a common concept, instead of sysbench concept.
2. Create it in the Disk message, if it doesn't exist. But not group them like "sysbench params".

Reply:
Will check on this